crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,17 @@ def addons
       end
     end
 
+    def repocheck
+      {
+        cloud: {
+          available: repo_version_available("suse-openstack-cloud", "7") == 0
```

Comment:
No idea. It's true we do not need to know, so maybe leave it there, but add a comment that it needs to be adapted.